### ISSUE TITLE
use bip39 for ledger/sync recovery words

### ISSUE THREAD

**Issue by diracdeltas** [brave/browser-laptop#13313]
see https://github.com/brave/browser-laptop/issues/13088 for how this should be implemented

cc @davidtemkin @bsclifton for triage

@bradleyrichter please attach your proposed design for the 16/24 word input

**Comment by bsclifton** [comment]
Assigning to 0.23.x to align with sync

**Comment by bsclifton** [comment]
Cezar, can you check this out? (to go along with the Sync work you've already done?)

Once a PR is ready, we can create a security review 😄

**Comment by cezaraugusto** [comment]
yes if that's an option I'd like to include this after sync v2 is merged as too many things have changed in there. It would speed-up both code and security review too IMO.

**Comment by bsclifton** [comment]
@NejcZdovc are the bip39 changes in a branch? If not, would we be able to create one? We are moving sync to 0.24.x in favor of Tor tabs
